clang/test/Driver/sycl-offload-new-driver.c

```diff
@@ -223,3 +223,8 @@
 // CHK-MACRO-GFX90A: clang{{.*}} "-fsycl-is-device"{{.*}} "-D__SYCL_TARGET_AMD_GPU_GFX90A__"{{.*}}
 // CHK-MACRO-GFX90C: clang{{.*}} "-fsycl-is-device"{{.*}} "-D__SYCL_TARGET_AMD_GPU_GFX90C__"{{.*}}
 
+/// Check that -sycl-device-link is passed to clang-linker-wrapper tool
+// RUN: %clangxx -fsycl -### --offload-new-driver \
+// RUN:          -fsycl-link %s 2>&1 \
```

Comment:
What is the behavior if we passed `-fsycl-link=bad_value` ?

Reply:
Thanks for the question.

I tried this. We get an error.
clang++: error: invalid argument 'bad_value' to -fsycl-link=

Also, -fsycl-link is not a new option that is being introduced here. So, testing its behavior is beyond the scope of this PR. I assume testing is already in place.

Thanks